cmd/explaintest/t/access_path_selection.test

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE `pay_daily` (
+`id` varchar(50) NOT NULL DEFAULT '',
+`a_id` bigint(11) NOT NULL DEFAULT '0',
+`b_id` bigint(11) NOT NULL DEFAULT '0',
+`time_stamp` bigint(11) DEFAULT NULL,
+`date_key` int(20) DEFAULT NULL,
+`io_type` int(11) DEFAULT NULL,
+`update_source` int(11) DEFAULT NULL,
+`update_time` timestamp DEFAULT NULL,
+PRIMARY KEY (`id`, `a_id`, `b_id`),
+KEY `IDX_ID` (`a_id`, `b_id`, `time_stamp`),
+KEY `IDX_DK` (`date_key`),
+KEY `IDX_TENANT_POI_DATEKEY` (`a_id`, `b_id`, `date_key`),
+KEY `IDX_US` (`update_source`, `a_id`)
+);
+explain select id, a_id, b_id, time_stamp, date_key, update_source from pay_daily where a_id=12565 and b_id=159298363 and date_key=20180731;
+explain select id from pay_daily where a_id=12565 and b_id=159298363 and time_stamp=10000;
+explain select id from pay_daily where update_source>12565 and a_id<159298363;
+explain select id from pay_daily where date_key>12565;
+explain select id from pay_daily where date_key>12565 and io_type=1;
+explain select id from pay_daily where date_key>12565 and a_id<159298363;
+
+CREATE TABLE `access_path_selection` (
+`a` int,
+`b` int,
+KEY `IDX_a` (`a`),
+KEY `IDX_b` (`b`),
+KEY `IDX_ab` (`a`, `b`)
+);
+insert into `access_path_selection` values (1, 0);
```

Comment:
Since it uses pseudo statistics. Do we need to insert values?

Reply:
OK, these useless insert statements have been removed.